.github/workflows/docs.yml

```diff
@@ -0,0 +1,40 @@
+name: Build and Deploy docs
+
+on:
+  push:
+    branches: [ main, jmm/sphinx-doc-skeleton ]
```

Comment:
```suggestion
    branches: [ main ]
```

Reply:
Should this be `develop` rather than `main` or does this not matter as `develop` is our default (`main`) branch?